Podcast/PlayerViewController.swift

```diff
@@ -180,4 +177,12 @@ class PlayerViewController: TabBarAccessoryViewController, PlayerDelegate, Playe
         Player.sharedInstance.setProgress(progress: Double(controlsView.slider.value))
     }
     
+    func playerControlsDidTapRecommendButton() {
```

Comment:
this is where you shoud implement that backend request I was talking about before